app/src/main/res/values/strings.xml

```diff
@@ -1,81 +1,179 @@
+    <!-- LOCALIZATION NOTE (keyboard_enter_label): The string 'keyboard_enter_label' is used on the
+         virtual keyboard to label the enter key. -->
     <string name="keyboard_enter_label">ENTER</string>
-    <!-- Parameter will be replaced with the app name -->
+    <!-- LOCALIZATION NOTE (permission_camera): '%1$s' will be replaced at run time with the app name.
+         The string 'permission_camera' will be displayed in a dialog box presented to the user
+         when the application needs to request permission to access the device's camera. -->
     <string name="permission_camera">Will you allow %1$s to use your camera?</string>
-    <!-- Parameter will be replaced with the app name -->
+    <!-- LOCALIZATION NOTE (permission_microphone): '%1$s' will be replaced at runtime with the app name.
+         The string 'permission_microphone' will be displayed in a dialog box presented to the user
+         when the application needs to request permission to access the device's microphone.-->
     <string name="permission_microphone">Will you allow %1$s to use your microphone?</string>
-    <!-- Parameter will be replaced with the app name -->
+    <!-- LOCALIZATION NOTE (permission_camera_and_microphone): '%1$s' will be replaced at runtime with the app name.
+         The string 'permission_camera_and_microphone' will be displayed in a dialog box presented to the user
+         when the application needs to request permission to access the device's camera and microphone. -->
     <string name="permission_camera_and_microphone">Will you allow %1$s to use your camera and microphone?</string>
-    <!-- Parameter will be replaced with the app name -->
+    <!-- LOCALIZATION NOTE (permission_location): '%1$s' will be replaced at runtime with the app name.
+         The string 'permission_location' will be displayed in a dialog box presented to the user
+         when the application needs to request permission to access the device's geographic location. -->
     <string name="permission_location">Will you allow %1$s to access your location?</string>
+    <!-- LOCALIZATION NOTE (permission_notification): '%1$s' will be replaced at runtime with the app name.
+         The string 'permission_notification' will be displayed in a dialog box presented to the user
+         when the application needs to request permission to send system notifications. -->
     <!-- Parameter will be replaced with the app name -->
     <string name="permission_notification">Will you allow %1$s to send notifications?</string>
-    <!-- Parameter will be replaced with the app name -->
+    <!-- LOCALIZATION NOTE (permission_read_external_storage): '%1$s' will be replaced at runtime with the app name.
+         The string 'permission_read_external_storage' will be displayed in a dialog box presented to the user
+         when the application needs to request permission to access the device's external storage such as the
+         sdcard. -->
```

Comment:
`SD card` or (`microSD` card)

Reply:
This is a little confusing as on android /sdcard isn't necessarily an actual SD card. It's just a shared disk partition.